alacritty/src/event.rs

```diff
@@ -456,13 +457,13 @@ impl<N: Notify> Processor<N> {
                 },
                 Event::ConfigReload(path) => {
                     processor.ctx.message_buffer.remove_target(LOG_TARGET_CONFIG);
-                    processor.ctx.resize_pending.message_buffer = Some(());
+                    processor.ctx.display_reconfig_pending.message_buffer = Some(());
 
                     if let Ok(config) = config::reload_from(&path) {
                         processor.ctx.terminal.update_config(&config);
 
-                        if *processor.ctx.font_size == processor.config.font.size {
-                            processor.ctx.resize_pending.font_size = Some(FontResize::Reset);
+                        if *processor.ctx.font != config.font {
```

Comment:
I do still need to think about how to properly address https://github.com/jwilm/alacritty/issues/2737#issuecomment-524929073 here, will work on that tonight or tomorrow morning.